src/main/java/com/softlayer/api/Mask.java

```diff
@@ -44,7 +45,26 @@ protected String getMask() {
 
     @Override
     public String toString() {
-        return toString(new StringBuilder()).toString();
+        String objectMask = new String();
+        String builtedMask = toString(new StringBuilder()).toString();
+        if(builtedMask.contains("[")){
+            String [] mask = builtedMask.split(Pattern.quote("["));
+            for (int count = 0; count < mask.length; count ++ ) {
+                if (count != 0){
```

Comment:
You could change the for loop to just start at 1 instead of 0, and remove this if statement